### ISSUE TITLE
I want to be able to load files from my iodide.io workspace or an export bundle with a "fetch" cell

### ISSUE THREAD

**Issue by bcolloran** [iodide-project/iodide#1009]
fetch cell will have syntax e.g.:

```
%% fetch
foo = data/filename.jpg // optional comment string
bar=   data/data.csv         // optional comment string
bar=data/data2.csv
```

evaling this should kick off a promise.all() that runs all the fetches in parallel, and further execution should wait until this eval completes. as soon as the fetches land, the raw content of the fetch should be assigned to the variable name

this will need to be smart enough to know when the notebook is in stndalone export mode, in which case it will need to use an alternative method of loading the files, as fetch does not work on the filesystem:
https://stackoverflow.com/questions/50007055/fetch-request-to-local-file-not-working

in the pathological (but it can happen) case of having a variable name repeated e.g.
```
%% fetch
bar = data/data.csv
bar = data/other_data.csv
```

we should notice this at parse time and only perform the final fetch that maps to the given var name

implementation detail:
if needed in order to deal with cross-origin problems with loading data from iodide.io or other places, the fetch cell in the eval-frame can send a signal to the editor asking the editor to make the fetch request; upon reciept of the data, the editor can hand off ownership of the object to the eval frame

(fyi @wlach  @hamilton -- did you have more thoughts on this?)

**Comment by wlach** [comment]
I like @hamilton's idea. Re: the local filesystem mode, I think we should deprecate support for that. It's a power user feature, and I don't think it's that unreasonable to ask people to just run `python -mhttp.server 8888` from their dev directory if they really want to use stuff from the file system.

**Comment by bcolloran** [comment]
hmm, i'm not sure overloading the import statement to work for things other than scripts is a road we want to go down ATM. I like what you described above, let's go for that.

**Comment by hamilton** [comment]
@bcolloran yeah I agree. Also it pretty closely ties fetching resources to javascript, which to me is the big deal-breaker.

**Comment by hamilton** [comment]
Closed by #1115
